HLTrigger/Configuration/python/CustomConfigs.py

```diff
@@ -31,6 +31,8 @@ def Base(process):
 #        process.GlobalTag.connect   = 'frontier://FrontierProd/CMS_CONDITIONS'
 #        process.GlobalTag.pfnPrefix = cms.untracked.string('frontier://Frontie#rProd/')
 #        
+    process.GlobalTag.snapshotTime = cms.string("9999-12-31 23:59:59.000")
```

Comment:
Do I understand it correctly that these customs never make it to relval
production workflows? Otherwise we can get conditions changing during
relval production.
On Aug 5, 2015 4:28 AM, "Martin Grunewald" <notifications@github.com> wrote:

> In HLTrigger/Configuration/python/CustomConfigs.py
> <https://github.com/cms-sw/cmssw/pull/10506#discussion_r36283767>:
>
> > @@ -31,6 +31,8 @@ def Base(process):
> >  #        process.GlobalTag.connect   = 'frontier://FrontierProd/CMS_CONDITIONS'
> >  #        process.GlobalTag.pfnPrefix = cms.untracked.string('frontier://Frontie#rProd/')
> >  #
> > +    process.GlobalTag.snapshotTime = cms.string("9999-12-31 23:59:59.000")
>
> No: this is for the updated L1 menus versions - we can't wait for the GT
> PRs,
> and some L1T menus will never make a GT (eg, the special LowPU L1T menus).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10506/files#r36283767>.
>